parametric/test_span_sampling.py

```diff
@@ -204,15 +196,14 @@ def test_single_rule_rate_limiter_span_sampling_sss008(test_agent, test_library)
     assert_sampling_decision_tags(span, sample_rate=None, mechanism=None)
 
     # wait a second for rate limiter tokens to replenish
-    time.sleep(1)
+    time.sleep(2)
```

Comment:
is this change intentional?

Reply:
Seems like yes: https://github.com/DataDog/system-tests/pull/576#issue-1411467823

Reply:
Yep. When I was testing it locally it was a bit flaky. It was giving 60% when it wanted 100%, probably because the sampling was done over a time period that may have overlapped here. Giving it an extra second for the rate limiter to catch up fixed the issue.